modules/react/modal/lib/ModalOverlay.tsx

```diff
@@ -4,81 +4,60 @@ import ReactDOM from 'react-dom';
 import {
   createSubcomponent,
   createElemPropsHook,
-  StyledType,
   useWindowSize,
   useForkRef,
-  getTheme,
 } from '@workday/canvas-kit-react/common';
 import {usePopupModel, usePopupStack} from '@workday/canvas-kit-react/popup';
 import {keyframes} from '@emotion/react';
-import styled from '@emotion/styled';
-import {Box, BoxProps} from '@workday/canvas-kit-react/layout';
+import {Box, BoxProps, mergeStyles} from '@workday/canvas-kit-react/layout';
 import {useModalModel} from './hooks';
+import {createStencil} from '@workday/canvas-kit-styling';
+import {system} from '@workday/canvas-tokens-web';
 
 export interface ModalOverlayProps extends BoxProps {}
 
-const fadeIn = keyframes`
-  from {
-    background: none;
-  }
-  to {
-    background: rgba(0,0,0,0.65);
-  }
-`;
+const fadeIn = () => {
+  return keyframes({
+    from: {
+      background: 'none',
+    },
+    to: {
+      background: `rgba(0,0,0, ${system.opacity.overlay})`,
+    },
+  });
+};
 
-const Container = styled(Box)<StyledType>(
-  {
+const container = createStencil({
+  base: {
     position: 'fixed',
-    top: 0,
-    left: 0,
+    top: system.space.zero,
+    left: system.space.zero,
     width: '100vw',
     height: '100vh',
-    background: 'rgba(0,0,0,0.65)',
-    animationName: `${fadeIn}`,
+    background: `rgba(0,0,0, ${system.opacity.overlay})`,
```

Comment:
I guess there is a color token for overlay

Reply:
yeah, it's `system.color.bg.overlay`

Reply:
Updated